content/ansible-tower-getting-started/7-wrap/_index.md

```diff
@@ -70,41 +68,39 @@ Only the part deploying the template is shown
 
 There is of course more then one way to accomplish this, but here is what you should do:
 
-- Make sure all hosts are in the inventory group `Webserver`.
+- Make sure all three hosts are in the inventory `Webserver`.
 
 - Define a variable `stage` with the value `dev` for the `Webserver` inventory:
 
-  - Add `stage: dev` to the inventory `Webserver` by putting it into the **VARIABLES** field beneath the three start-yaml dashes. Click **SAVE**
+  - Add `stage: dev` to the inventory `Webserver` by putting it into the **Variables** field beneath the three start-yaml dashes.
 
 {{% notice warning %}}
-Make sure to add the variable to the inventory and **not** to the new node3!
+Make sure to add the variable to the inventory and **not** to a node! And take care to keep the three dashes that mark the YAML start in place when adding variables in Automation Controller!
```

Comment:
the three dashes are actulaly automatically removed after you click save

Reply:
Sentence removed